{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--c9d8f92c-0096-4fea-8da9-d8cf9bc27258",
    "created": "2024-08-13T16:06:04.161587Z",
    "modified": "2024-08-13T16:06:04.161591Z",
    "name": "No Title Available",
    "description": "A flaw was found in privoxy before 3.0.32. A crash may occur due a NULL-pointer dereference when the socks server misbehaves.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2021-20274"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://www.privoxy.org/announce.txt"
        },
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1936662"
        },
        {
            "name": "GLSA-202107-16",
            "tags": [
                "vendor-advisory",
                "x_refsource_GENTOO"
            ],
            "url": "https://security.gentoo.org/glsa/202107-16"
        }
    ]
}